misc/README.md

```diff
@@ -28,7 +28,11 @@ NOTE: This has only been tested on Linux. It may or may not work on OSX/other en
 
 The release tester is a script to automate running Wallaroo Up and all example applications for every distribution supported by Wallaroo Up. It does so by looping through a list of supported distributions and then using either Vagrant or Docker to start the appropriate environment for running Wallaroo Up and the Wallaroo example tester.
 
-The release tester requires an environment with Docker and/or Vagrant/Virtualbox installed and available with Docker set up to be run without `sudo`.
+The release tester requires an environment with Docker and/or Vagrant/Virtualbox installed and available with Docker set up to be run without `sudo`. Vagrant also requires the `vagrant-vbguest` plugin to be install. It can be installed via the following command:
```

Comment:
I was running the example tester with a docker that *did* require sudo. I didn't run into any problems...

Reply:
That's interesting... The example test script doesn't run the example commands using `sudo`. This should normally result in an error for `docker` when running the `kafka` examples.

One possible reason might be if you ran it using `sudo`... i.e. `sudo ./example-tester.sh`. Is that what you did?

Reply:
No I didn't...

Reply:
Weird, I'm fairly certain I added the `sudo` note because I ran into an error at some point. Maybe some day we'll solve this mystery. Thanks for letting me know if works without the docker sudo thing. I'll leave that note in the README for now but we might want to remove it for a future update once at least one other person confirms your experience.

Reply:
okay